certified-connectors/RecordedFutureNew/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1575 @@
+{
+    "swagger":"2.0",
+    "info":{
+        "title":"Recorded Future New",
+        "description":"Recorded Future Connector enables access to the Recorded Future Intelligence. The connector has dedicated actions for pulling Recorded Future indicators (IP, Domain, URL, Hash) and associated context (Risk Score, Risk Rules, Intelligence Card Link and High Confidence Evidence Based Links), Vulnerabilities, Recorded Future Alerts and enables access to Recorded Future SOAR API and Fusion Files",
+        "contact":{
+            "name":"Recorded Future Support",
+            "url":"https://support.recordedfuture.com",
+            "email":"support@recordedfuture.com"
+        },
+        "version":"1.0"
+    },
+    "host":"api.recordedfuture.com",
+    "basePath":"/gw/azure",
+    "schemes":[
+        "https"
+    ],
+    "consumes":[
+        
+    ],
+    "produces":[
+        
+    ],
+    "paths":{
+        "/lookup/ip/{ip}":{
+            "get":{
```

Comment:
@johannesh-recordedfuture Indentations - The properties are not aligned properly and there is a longer tab index between the properties in the swagger and apiProperties.json files. Please refer to other certified connector artifacts in the repo.